config/jobs/kubernetes/sig-cloud-provider/gcp/gcp-gce.yaml

```diff
@@ -124,6 +124,58 @@ presubmits:
         securityContext:
           privileged: true
 
+  - name: pull-kubernetes-e2e-gce-graceful-shutdown
+    always_run: false
+    cluster: k8s-infra-prow-build
+    skip_branches:
+    - release-\d+\.\d+ # per-release image
+    annotations:
+      testgrid-dashboards: presubmits-kubernetes-nonblocking
+      testgrid-tab-name: pull-kubernetes-e2e-gce-graceful-shutdown
+    labels:
+      preset-service-account: "true"
+      preset-k8s-ssh: "true"
+      preset-dind-enabled: "true"
+      preset-pull-kubernetes-e2e: "true"
+      preset-pull-kubernetes-e2e-gce: "true"
+    decorate: true
+    decoration_config:
+      timeout: 80m
+    path_alias: k8s.io/kubernetes
+    extra_refs:
+    - org: kubernetes
+      repo: release
+      base_ref: master
+      path_alias: k8s.io/release
+    spec:
+      containers:
+      - command:
+          - runner.sh
+          - /workspace/scenarios/kubernetes_e2e.py
+        args:
+        - --build=quick
+        - --cluster=
+        - --env=NODE_KUBELET_TEST_ARGS=--shutdown-grace-period=30s --shutdown-grace-period-critical-pods=10s
```

Comment:
per discussion offline it seems we need to use the config file https://kubernetes.io/blog/2021/04/21/graceful-node-shutdown-beta/#how-do-i-use-it

Reply:
Indeed, I've also added that context to the PR description at https://github.com/kubernetes/kubernetes/pull/125413 and revived it to set these Kubelet parameters via the config file for linux nodes.